### PR TITLE
3.0 - make I18N shells extract task differentiate similar msgs with different contexts

### PR DESCRIPTION
**Pull request by PGBI**
This PR aims at fixing the issue #6542

### CODE REVIEW

File: src/Shell/Task/ExtractTask.php [hunk 1]
```diff
@@ -449,35 +444,36 @@ protected function _buildFiles()
...
+                foreach ($contexts as $context => $details) {
+                    $plural = $details['msgid_plural'];
+                    $files = $details['references'];
+                    $occurrences = [];
+                    foreach ($files as $file => $lines) {
+                        $lines = array_unique($lines);
+                        $occurrences[] = $file . ':' . implode(';', $lines);
+                    }
+                    $occurrences = implode("\n#: ", $occurrences);
+                    $header = '#: ' . str_replace(DS, '/', str_replace($paths, '', $occurrences)) . "\n";
```
Comment (lorenzo) [inline src/Shell/Task/ExtractTask.php]: Did you change from spaces to tabs?

Comment (markstory) [inline src/Shell/Task/ExtractTask.php]: It looks like the block was indented to accomodate the additional loop.

**Comment by markstory** [comment]
Looks good to me.

File: src/Shell/Task/ExtractTask.php (hunk 1 above)
Comment (lorenzo) [inline src/Shell/Task/ExtractTask.php]: Ah, I get it now

**Comment by lorenzo** [comment]
Thanks @PGBI !

**Pull request merged**